File: Assignments/Week_01/primeGenerator.js
const isPrime = (num) => {
	if (num >= 1 && num <= 3)
		return true
	else if (num % 2 === 0 || num % 3 === 0)
		return false
	let i = 5
	while( i*i <= num){
		if (num % i === 0 || num % (i+2) === 0){
			return false
		}
		i += 6

	}
	return true
}


const primeGenerator = (n) => {
	count = 0 
	num = 0
	while(count < n){
		if(isPrime(num)){
			console.log(num)
			count += 1
		}
		num +=1
	}
}

primeGenerator(26)

File: Assignments/Week_01/primeGenerator.py
def isPrime(num):
	if (num >= 1 and num <= 3):
		return True
	elif (num % 2 == 0 or num % 3 == 0):
		return False
	i = 5
	while (i*i <= num):
		if (num % i == 0 or num % (i+2) == 0):
			return False
		i += 6
	return True

def generatePrime(n):
	i = 0
	while (i < n):
		if (isPrime(i)):
			print(i)
		i += 1

generatePrime(25)
			